internal/services/network/express_route_connection_resource.go

```diff
@@ -183,6 +189,7 @@ func resourceExpressRouteConnectionCreate(d *pluginsdk.ResourceData, meta interf
 			RoutingConfiguration:      expandExpressRouteConnectionRouting(d.Get("routing").([]interface{})),
 			RoutingWeight:             utils.Int32(int32(d.Get("routing_weight").(int))),
 			ExpressRouteGatewayBypass: utils.Bool(d.Get("express_route_gateway_bypass_enabled").(bool)),
+			PrivateLinkFastPath:       utils.Bool(d.Get("private_link_fast_path_enabled").(bool)),
```

Comment:
We should add some validation here e.g. 

```
	if d.Get("private_link_fast_path_enabled").(bool) && !d.Get("express_route_gateway_bypass_enabled").(bool) {
		return fmt.Errorf("`express_route_gateway_bypass_enabled` must be enabled when `private_link_fast_path_enabled` is set to `true`")
	}
```

Reply:
I've tried my best putting that at the right position, which should be right before we set the parameters. I've also added it for both the `Create` and `Update` function, so we keep consistency.